### PR TITLE
Add correct architecture blurb

### PR DESCRIPTION
**Pull request by 8W9aG**

### CODE REVIEW

**Comment by 8W9aG** [comment]
@spotify/objc-dev

File: README.md [hunk 1]
```diff
@@ -15,7 +15,11 @@ Everyone tries to implement a cache at some point in their app’s lifecycle, an
 - [x] 💻 OS X 10.9+
 
 ## Architecture :triangular_ruler:
-`SPTPersistentCache` is designed as an LRU cache which stores all the data in a single binary file, with entries containing the length, last accessed time and a CRC check designed to prevent corruption. It can be used to automatically schedule garbage collection and invoke pruning.
+`SPTPersistentCache` is designed as an LRU cache which makes use of the file system to store files as well as inserting a cache header into each file at its head. This cache header allows us to track the TTL, last updated time, the redundancy check and more. This allows the cache to know how often a file is accessed, when it was made, whether it has become corrupt and allows decisions to be made on whether the cache is stale.
```
Comment (ChocoChipset) [inline README.md]: > as well as inserting a cache header into each file at its head

If it's a header isn't it by default in the _head_?

**Comment by ChocoChipset** [comment]
:+1:

**Comment by rastersize** [comment]
:+1:

**Pull request merged**